### ISSUE TITLE
Looking for contributors!

### ISSUE THREAD

**Issue by IvanJosipovic** [IvanJosipovic/BlazorTable#330]
Hi everyone,

Life has gotten rather busy over the last year and I don't have as much time as I used to continue enhancing this library. Myself and many contributors have spent a considerable amount of time to get it this far and would love to see it continue.

I'm hoping that we can get a few people that are interesting in working with me and continue supporting this great library.

What I will commit to:
- Walking the contributors through the current design
- Assist with major changes
- Assist with roadmap
- Maintain fully automated CI/CD with beta and production release support
  - I will simplify the current CI/CD pipeline and enable automatic versioning with [Semantic Releases](https://github.com/semantic-release/semantic-release) and will add a [PR bot](https://github.com/zeke/semantic-pull-requests) which will make sure all PRs have associated tags

Pinging existing contributors but this offer is open to anyone.
@conficient @NilsPur @carlos-sergio-carvalho @pqbui001 @nadais @chrgraefe @leonardder @KBBA @ekuhlmann23 @smokedlinq @NetherGranite

Thank you!

**Comment by conficient** [comment]
Yes I'm willing to contribute. On holiday right now so will be more available next week

**Comment by conficient** [comment]
I'm starting to look at issues and PRs. I'll prioritize PRs since they have put more effort and should be easier to clear down. Any I'm not sure about I'll ping you first to discuss.

**Comment by IvanJosipovic** [comment]
@conficient, thank you for your help.

Please familiarize yourself with https://www.conventionalcommits.org/en/v1.0.0/
This will enable automatic package versioning, and prevent the PR bot from blocking a PR.

Here are some example commit messages:
- chore: updated readme
  - This will not trigger a new release
- fix: resolved bug #0000
  - This will increment the patch version 1.0.X
- feat: added X feature
  - This will increment the minor version 1.X.0
- feat! or fix!: added new breaking feature
  - This will increment the major version X.0.0
- BREAKING CHANGE: can be used in combination with feat/fix etc to increment the major version X.0.0

**Comment by conficient** [comment]
Thanks for the pointers.. I've not used this before but might have to add to my own projects now.. :)
